packages/ui5-application-writer/templates/optional/codeAssist/package.json

```diff
@@ -3,5 +3,8 @@
         "eslint": "7.32.0",
         "@sap/eslint-plugin-ui5-jsdocs": "2.0.5",
         "@sapui5/ts-types": "1.102.x"
+    },
+    "scripts": {
+        "lint": "eslint webapp/ --ext .js,.ts"
```

Comment:
We currently write a lint script like `"lint": "eslint ./ --rulesdir ./node_modules/eslint-plugin-fiori-custom/lib/rules/",`
We have a TBI to modify `eslint-plugin-fiori-custom` to work better with the format `eslint webapp/ --ext .js,.ts`.
I think it is probably better to hold off for now this change.

Reply:
The script comes when being integrated in the SAP Fiori tools, correct? So, it would overwrite the default script, however, for other consumers, at least a basic script would be there.
At least, that was my thinking, was I wrong?

Reply:
Sorry I am mixing up eslint and code assist options. 
Yeah I think this is ok.

Reply:
Im also confused by this. We are adding a lint script as part of code assist option but we also have an eslint option. Shouldn't these be kept as discreet options (if you want to combine them during writing then fine, but its unexpected to see this here under `./codeAssist`)

Reply:
You are completely right. I mixed something up here. Will be fixed.

Reply:
Fixe with https://github.com/SAP/open-ux-tools/pull/667/commits/e9a36707461c5e0416695c6e26bbb7a79d1aa545